### PR TITLE
[Mobile app] Enhancing the feedback feature

### PR DESCRIPTION
**Pull request by NoahNsimbe**
#### Summary of Changes (What does this PR do?)

- [x] Enhances state management for feedback page.
- [x] Auto fills email address for users when sign in using their email address.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [MOB-127](https://airqoteam.atlassian.net/browse/MOB-127)

#### Screenshots (optional)

### CODE REVIEW

File: mobile/lib/blocs/feedback/feedback_bloc.dart [hunk 1]
```diff
@@ -1,221 +1,176 @@
...
+    }
+
+    if (state.blocStatus == BlocStatus.success) {
+      return emit(
+        FeedbackState.initial().copyWith(
+          emailAddress: profile.emailAddress,
+          feedbackChannel: FeedbackChannel.email,
+        ),
+      );
+    }
```
Comment (NoahNsimbe) [inline mobile/lib/blocs/feedback/feedback_bloc.dart:40]: Setting default value for email address incase the user signed in using their email address.

Comment (NoahNsimbe) [inline mobile/lib/blocs/feedback/feedback_bloc.dart:40]: Setting default value for email address incase the user signed in using their email address.

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**